ci/pipelines/deploy-to-test.yml

```diff
@@ -2650,34 +2667,50 @@ jobs:
           AWS_ACCESS_KEY_ID: ((.:role.AWS_ACCESS_KEY_ID))
           AWS_SECRET_ACCESS_KEY: ((.:role.AWS_SECRET_ACCESS_KEY))
           AWS_SESSION_TOKEN: ((.:role.AWS_SESSION_TOKEN))          
-      - put: products-ui-latest-ecr-registry-test
-        params:
-          image: products-ui-candidate-ecr-registry-test/image.tar
-#    on_failure:
-#      put: slack-notification
-#      attempts: 10
-#      params:
-#        channel: '#govuk-pay-announce'
-#        silent: true
-#        text: ':red-circle: products-ui candidate image ((.:candidate_image_tag)) failed post-merge end to end tests - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>'
-#        icon_emoji: ":concourse:"
-#        username: pay-concourse
-#    on_success:
-#      put: slack-notification
-#      attempts: 10
-#      params:
-#        channel: '#govuk-pay-activity'
-#        silent: true
-#        text: ':green-circle: products-ui candidate image ((.:candidate_image_tag)) passed post-merge end to end tests - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>'
-#        icon_emoji: ":concourse:"
-#        username: pay-concourse       
+      - in_parallel:
+        - do:
+          - put: products-ui-ecr-registry-test
+            params:
+              image: products-ui-candidate/image.tar
+              additional_tags: parse-candidate-tag/release-tag
+            get_params:
+              skip_download: true
+          - put: products-ui-latest
+            params:
+              image: products-ui-candidate/image.tar
+            get_params:
+              skip_download: true
+        - put: products-ui-dockerhub
```

Comment:
Is putting to dockerhub temporary?

Reply:
That's a good existential question :)

Unless we reqiure devs to auth to aws test and update our docker-compose files to pull from there then it's perm.

However I think we should probably move in the future to defaulting to ecr and not putting them in dockerhub at all.

At least that's a trivial change unlike adding all these :D

Reply:
But what's the purpose of putting it in dockerhub now? Since we're removing e2e from jenkins.

Reply:
I'd _love_ to be able to do away with Docker Hub in the future. Hopefully this will actually be possible and we could bake in some kind of helper into Pay CLI to help automate the authentication process.

Reply:
To allow developers to still run the e2e tests locally

Reply:
Sadly unless we write a whole new section for pay-cli it wont help with end to end tests, only with pay-local (which we also have to account for, and is another reason to push to dockerhub still)

Reply:
Oh yeah pay local...